Jenkinsfile

```diff
@@ -34,6 +34,43 @@ pipeline {
         ], wait: false
       }
     }
+
+    stage('Deploy dev and staging') {
+      when { branch 'master' }
```

Comment:
Not to overcomplicate this, but we occasionally have need to change what branch gets deployed particularly to dev. E.g. we'll want to have a disruptive feature branch on dev for a week. I think we'd have to just copy/paste this with a different `when` expression and split out the dev vs. staging deploy jobs, which is not too onerous, but if you see something more clever to do here keep that requirement in mind.

Reply:
Hmm. The primary reason that the jobs get triggered here are to ensure the correct git hash is being sent to the build and deploy jobs (Jenkins doesn't appear to be able to query a job for specific info to trigger it on the downstream job and query this one).

We could potentially drop the deploy from here and add a meta job that watches master for commits and does this log there, but then we'd lose testing the specific commit first.